index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link rel="stylesheet" href="tributePage.css">
+</head>
+<body>
+    <header class="Alex">
+        <h1>Dr. Jane Goodall</h1>
+        <p>The Woman who saved billions of life</p>
+    </header>
+    <main id="main">
+        <div id="img-div">
+        <img id="image" src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/84/JaneGoodallOct10.jpg/330px-JaneGoodallOct10.jpg" alt="">
+        <p>Dr. Jane Goodall, trians biologists in italy on how to increase 
+            wheat yields - part of her life-long war on hunger</p>
+    </div>
+
+        <h2 id="tribute-info">Biography</h2>
+        <div class="longList">
+        <ul>
+            <li>1914 - Born in Cresco, Iowa</li>
+            <li>1933 - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program 
+                known as the "National Youth Administration"</li>
+            <li>1935 - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+            <li>1937 - Finishes university and takes a job in the US Forestry Service
+            </li>
+            <li>1938 - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.
+            </li>
+            <li>1941 - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science.
+            </li>
+            <li>1942 - Receives a Ph.D. in Genetics and Plant Pathology
+            </li>
+            <li>1944 - Rejects a 100% salary increase from Dupont, leaves behind his pregnant wife, and flies to Mexico to head a new plant pathology program. Over the next 16 years, his team breeds 6,000 different strains of disease resistent wheat - including different varieties for each major climate on Earth.
+            </li>
+            <li>1945 - Discovers a way to grown wheat twice each season, doubling wheat yields
+            </li>
+            <li>2009 - dies at the age of 95.
+            </li>
+    
+        </ul>
```

Comment:
Please conform to a single code style, helps code legibility.

Reply:
Good job by telling the student **WHAT** he should change but also explain to him **WHY** he needs to change that thing. It might be obvious to you but there's a chance that others won't be thinking in that way. That's why it's better to also explain the reason why he should make the changes. 😄 👍🏻 

You can refer him to some cool extensions for formatting the code. For example `Prettier` extension in VS Code for formatting the code. 👍🏻

Reply:
Good job by telling the student WHAT he should change but also explain to him WHY he needs to change that thing. It might be obvious to you but there's a chance that others won't be thinking in that way. That's why it's better to also explain the reason why he should make the changes. 😄 👍🏻

You can refer him to some cool extensions for formatting the code. For example Prettier extension in VS Code for formatting the code. 👍🏻